flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/planner/operations/SqlToOperationConverter.java

```diff
@@ -186,6 +204,64 @@ private Operation convertUseCatalog(SqlUseCatalog useCatalog) {
 		return new UseCatalogOperation(useCatalog.getCatalogName());
 	}
 
+	/** Convert use database statement. */
+	private Operation convertUseDatabase(SqlUseDatabase useDatabase) {
+		return new UseDatabaseOperation(useDatabase.fullDatabaseName());
```

Comment:
To be consistent with other db related operations, I think we can also try to resolve missing catalog name here.